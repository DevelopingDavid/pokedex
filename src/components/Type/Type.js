import React from 'react';

const Type = ({type}) => {
  return (
    <div className='type-container'>
      <h1 className={type}>{type}</h1>
    </div>
  )
};

export default Type;