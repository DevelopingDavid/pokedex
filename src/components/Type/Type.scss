.type-container {
  font-family: "Flexo-Medium",arial,sans-serif;
  border-radius: 3px;
  line-height: 18px;
  width: 110px;
  font-size: 13px;
  text-align: center;
  margin: 5px 3px 5px 0;
  color: #ffffff;
}

.grass {
  border-radius: 3px;
  background: #9ACC50;
}

.poison {
  border-radius: 3px;
  background: #B97FC8;
}

.fire {
  border-radius: 3px;
  background: #FD7D22;
}

.flying {
  border-radius: 3px;
  background: #BDB9B8;
}

.water {
  border-radius: 3px;
  background: #4592C3;
}

.bug {
  border-radius: 3px;
  background: #729F3E;
}

.normal {
  border-radius: 3px;
  background: #A3ACAE;
}

.electric {
  border-radius: 3px;
  background: #EED535;
}

.ground {
  border-radius: 3px;
  background: #AB9841;
}

.fairy {
  border-radius: 3px;
  background: #FDB9E9;
}

.fighting {
  border-radius: 3px;
  background: #D46722;
}

.psychic {
  border-radius: 3px;
  background: #F366B9;
}

.ground {
  border-radius: 3px;
  background: #A38B20;
}

.rock {
  border-radius: 3px;
  background: #A38B20;
}

.steel {
  border-radius: 3px;
  background: #9FB7B7;
}

.ice {
  border-radius: 3px;
  background: #51C4E7;
}

.ghost {
  border-radius: 3px;
  background: #7B62A3;
}

.dragon {
  border-radius: 3px;
  background: #F16E58;
}

