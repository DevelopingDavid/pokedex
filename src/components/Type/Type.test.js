import React from 'react';
import { shallow } from 'enzyme';
import Type from './Type';

const mockType = 'fire';

describe('Type', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Type type={mockType}/>
    )
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});