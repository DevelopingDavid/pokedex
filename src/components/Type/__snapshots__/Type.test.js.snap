// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Type should match snapshot 1`] = `
<div
  className="type-container"
>
  <h1
    className="fire"
  >
    fire
  </h1>
</div>
`;
